.travis/deploy_sbt.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o verbose
+
+# If we're on the master branch and in an application project, we should
+# build a new Docker image and push it to ECR.
+
+if [[ "$BRANCH" != "master" ]]
+then
+  echo "Not on master (BRANCH=$BRANCH); skipping deploy..."
+  exit 0
+fi
+
+if [[ "$PROJECT" == "common" ]]
+then
+  echo "Common lib doesn't have a Docker container; skipping deploy..."
+  exit 0
+fi
+
+export VERSION="0.0.1"
+export BUILD_ENV="prod"
```

Comment:
I might use bash’s ability to use variable defaults here:

```bash
export VERSION="${VERSION:-0.0.1}"
export BUILD_ENV="${BUILD_ENV:-dev}"
```

So this can be used to do dev deploys if necessary.